### PR TITLE
Add error codes to service commands

### PR DESCRIPTION
**Pull request by edaena**
Related to https://github.com/microsoft/bedrock/issues/1308

### CODE REVIEW

File: src/commands/service/create-revision.ts [hunk 1]
```diff
@@ -69,12 +71,10 @@ export const getDefaultRings = (
...
   if (defaultRings.length === 0) {
-    throw Error(
-      `Default branches/rings must either be specified in ${join(
-        __dirname,
-        "bedrock.yaml"
-      )} or provided via --target-branch`
-    );
+    throw buildError(errorStatusCode.VALIDATION_ERR, {
+      errorKey: "service-create-revision-cmd-err-default-branch-missing",
+      values: [join(__dirname, "bedrock.yaml")],
```
Comment (dennisseah) [inline src/commands/service/create-revision.ts:76]: i believe `join` should be `path.join`

File: src/lib/i18n.json [hunk 2]
```diff
@@ -247,6 +247,11 @@
     "project-create-variable-group-cmd-err-values-missing": "Required values were missing. Provide values for registry-name, hld-repo-url, service-principal-id, service-principal-password, tenant.",
     "project-init-cmd-failed": "Project init was not successfully executed.",
 
+    "service-create-revision-cmd-failed": "Service create revision was not successfully executed.",
+    "service-create-revision-cmd-err-pr-source-branch": "A pull request for a branch cannot be made against itself. Ensure your target branch(es) '{0}' do not include your source branch '{1}'",
```
Comment (dennisseah) [inline src/lib/i18n.json:251]: cannot -> could not

File: src/commands/service/create-revision.ts [hunk 3]
```diff
@@ -167,17 +168,22 @@ export const execute = async (
...
 
     await makePullRequest(defaultRings, values);
     await exitFn(0);
   } catch (err) {
-    logger.error(err);
+    logError(
+      buildError(
+        errorStatusCode.CMD_EXE_ERR,
+        "service-create-revision-cmd-failed",
+        err
```
Comment (dennisseah) [inline src/commands/service/create-revision.ts:184]: 👍

File: src/commands/service/create-revision.ts (hunk 1 above)
Comment (bnookala) [reply src/commands/service/create-revision.ts:76]: @dennisseah it's imported above from the module:

line 2:
```import { join } from "path";```

**Review by bnookala — APPROVED** [review]

File: src/commands/service/create-revision.ts (hunk 1 above)
Comment (dennisseah) [reply src/commands/service/create-revision.ts:76]: got it thanks

**Review by dennisseah — APPROVED** [review]

**Pull request merged**